Common/cpp/SharedItems/Shareables.h

```diff
@@ -265,13 +265,20 @@ class ShareableRemoteFunction
       public std::enable_shared_from_this<ShareableRemoteFunction> {
  private:
   jsi::Runtime *runtime_;
+#ifndef NDEBUG
```

Comment:
why not just `#ifdef DEBUG`?

Reply:
More details in PR description in #5113.

Reply:
Ah, interesting! 

![image](https://github.com/software-mansion/react-native-reanimated/assets/15199031/f2f33ef2-fcf2-46c8-a061-e8e85956f693)